### DISCUSSION TITLE
[ FF7 ] Oops, error.

### DISCUSSION THREAD

**Discussion by VelvetKusanagi** [julianxhokaxhiu/FFNx#199] [Q&A]
Probably seen this a thousand times, but I got an emergency crash and wondering how to fix it.

Link to files. Would like help with solution please.

I am using the Steam version. 
https://drive.google.com/drive/folders/1GUpANlFDtHsJHHnIkOYXiXMPFjkWqw4l?usp=sharing

Using 7H and I don't think it's the problem.

**Comment by julianxhokaxhiu** [comment]
Hi,

Your issue is totally unrelated to FFNx and because of this I moved it to a discussion. The crash you have is because of 7th, or actually because of some mods you're using which are not compatible, despite you seeing the error message coming out of FFNx ( which is misleading ).

Feel free to ask for support on the relative 7th support channels regarding your issue.

**Reply by VelvetKusanagi** [reply]
Thank you for the quick response. I will ask in 7H support.
